tests/storage/concurrent_vm_boot/utils.py

```
@@ -0,0 +1,65 @@
+    return dv.res
+
+
+def create_vm_with_disks(
+    index: int,
+    namespace_name: str,
+    client: DynamicClient,
+    storage_class_name: str,
+) -> VirtualMachineForTests:
+    """Create a VM with a container disk boot volume and NUM_BLANK_DISKS_PER_VM blank disks."""
+    vm_name = f"concurrent-vm-{index}"
+    LOGGER.info(f"Creating VM {vm_name}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)